var/spack/repos/builtin/packages/py-pydv/package.py

```diff
@@ -7,13 +7,22 @@
 
 
 class PyPydv(PythonPackage):
-    """PDV is a 1D graphics and data analysis tool, heavily based on the
-    ULTRA plotting tool"""
+    """PyDV is a 1D graphics and data analysis tool, heavily based on the ULTRA plotting tool."""
 
     homepage = "https://github.com/griffin28/PyDV"
-    url = "https://github.com/griffin28/PyDV/archive/pydv-2.4.2.tar.gz"
+    # url = "https://github.com/griffin28/PyDV/archive/pydv-3.1.15.tar.gz"
+    pypi = "llnl-pydv/llnl_pydv-3.1.15.tar.gz"
```

Comment:
[pydv](https://pypi.org/project/pydv/) and [llnl-pydv](https://pypi.org/project/llnl-pydv/) are two different packages. I agree with moving to the PyPI source file. But I think we want to create a new package.

Reply:
They are the same package, pydv was already taken on pypi, hence llnl-pydv. The same goes for ibis. I'm assuming you want the package under py-llnl-pydv then, instead of py-pydv?

Reply:
Correct. We try to match the names on PyPI.